content/ja/_index.html

```diff
@@ -41,12 +41,12 @@ <h2>150以上のマイクロサービスアプリケーションをKubernetes上
         <button id="desktopShowVideoButton" onclick="kub.showVideo()">ビデオを見る</button>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna20" button id="desktopKCButton">2020年11月17日-20日のKubeCon NAバーチャルに参加する</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna22" button id="desktopKCButton">2022年10月24日から28日までミシガン州のKubeConデトロイトに参加する</a>
         <br>
         <br>
         <br>
         <br>
-        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccnceu21" button id="desktopKCButton">2021年5月4日-7日のKubeCon EUバーチャルに参加する</a>
+        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccnceu22" button id="desktopKCButton">スペインのKubeConバレンシアに参加+バーチャル、2022年5月16〜20日</a>
```

Comment:
```suggestion
        <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/?utm_source=kubernetes.io&utm_medium=nav&utm_campaign=kccncna22" button id="desktopKCButton">2022年10月24日-28日のKubeCon NAバーチャルに参加する</a>
```

Reply:
To keep consistency with the previous events.

https://kubernetes.io/ja/

Reply:
@ptux I guess you need to approve the changes then only the bot can merge it. 😊

Reply:
@krsakshi 

Please be patient.
I'm not an approver, so I can't approve PRs.

https://kubernetes.io/docs/contribute/participate/roles-and-responsibilities/#approvers